com.chartboost.helium/Android~/helium-android-bridge/src/main/kotlin/com/chartboost/heliumsdk/unity/IBannerEventListener.kt

```diff
@@ -0,0 +1,16 @@
+package com.chartboost.heliumsdk.unity
+
+/**
+ * Interface for sending real-time Banner events
+ */
+interface IBannerEventListener {
+    fun DidLoadBanner(placementName: String?, errorCode: Int, errorDescription: String?)
```

Comment:
on the native side, placementName is not null, so that might make it easier for you.

Reply:
Yes, fixed it.